src/components/BlogPost.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import '../BlogPost.css' //allows us to use the css from our other file
+
+/***********************************************************************************************/
+
+//JavaScript Lambda Functions: these unique types allow functions to be passed around like data
+
+//boldIt: String -> Void (we take in a string and do something)
+//purpose: to bold the heading of the textarea a user is currently typing in
+const boldIt = (id: string) => {
+    document.getElementById(id)?.setAttribute("style", "font-weight: bold;")
+}
+
+//revert: String -> Void (we take in a string and do something)
+//purpose: to normalize the heading of the textarea a user is just clicked out of
+const revert = (id: string) => {
+    document.getElementById(id)?.setAttribute("style", "font-weight: normal;")
+}
+
+/***********************************************************************************************/
+
+//JavaScript Function: regular old functions that do something
+
+//submitIt: -> Event(we take in a nothing and return an event)
+//purpose: alert that we have submitted a post
+function submitIt() {
+    alert('You have just submitted a post!');
+}
+
+//cancelIt: -> Event(we take in a nothing and return an event)
+//purpose: alert that we have cancelled a post
+function cancelIt() {
+    alert('You have just cancelled a post!');
+}
+
+/**********************************************************************************************/
+
+//HTML/CSS
+
+const BlogPost: React.FC = () => {
```

Comment:
Can you rename this to be more descriptive of what it's doing? This is creating a blog post so I think it might help to have that in the component name.